File: MODUL 4/Soal 5/PRAK405-2410817210026-Fadhil_Syahdama_Mahatma_Putra.c
#include <stdio.h>
int main(){

int zani,bocchi;
int total=0;
scanf("%d %d",&zani, &bocchi);
for(int i=1; i<=zani; i++){
    int hasil=0;

    for(int j=i;j>=1; j--){
        hasil+=j*bocchi;
        printf("(%d*%d)",j,bocchi);
        if(j>1){
            printf(" + ");
        } }
printf("= %d\n",hasil);
total+=hasil;}
printf("%d",total);
}

File: MODUL 4/Soal 5/PRAK405-2410817210026-Fadhil_Syahdama_Mahatma_Putra.py
zani, bocchi = map(int, input().split())

total = 0

for i in range(1, zani + 1):
    hasil = 0

    for j in range(i, 0, -1):
        hasil += j * bocchi
        print(f"({j}*{bocchi})", end="")
        if j > 1:
            print(" + ", end="")

    print(f" = {hasil}")
    total += hasil
print(total)